FrontEnd/src/components/SignUp/components/signup/signup-form/RulesModal.css

```diff
@@ -1 +1,26 @@
-/* rules */
\ No newline at end of file
+.rules-modal {
+  position: absolute;
+  top: 60%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 500px;
+  max-height: 10vh;
+  overflow-y: auto;
+  padding: 20px;
+  background-color: white;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+}
+
+.rules-modal__title {
+  margin-bottom: 20px;
+}
+
+.rules-modal__button {
+  cursor: pointer;
+  background-color: #1f9a7c;
+  color: white;
+  border: none;
```

Comment:
Do we need it. It`s none by default

Reply:
This is an example of design; there is a button with a border and a white background. This is a sample.

Reply:
We don`t have 'make sample` task.

Reply:
Done